src/hip/handlehip.cpp

```diff
@@ -58,30 +58,11 @@
 #include <chrono>
 #include <thread>
 
-#define MIOPEN_WORKAROUND_ROCM_COMPILER_SUPPORT_ISSUE_30 (MIOPEN_USE_COMGR && BUILD_SHARED_LIBS)
```

Comment:
Is it so that we no longer need a workaround for https://github.com/RadeonOpenCompute/ROCm-CompilerSupport/issues/30 ?

Reply:
[process] @averinevg Let me "resolve" my review comments myself (unless the case is trivial), thanks!